js/core/component.js

```diff
@@ -504,12 +477,22 @@ var Component = Class.inherit({
         return Object.prototype.hasOwnProperty.call(deprecatedOptions, name);
     },
 
-    _setOptionSilent: function(name, value) {
+    _setOptionWithoutNotify: function(name, value) {
```

Comment:
I suppose this renaming and adding another method wtih the old name could confuse our collegues. Maybe it's better to find another names for the new methods or use the `silent` flag as argument?